src/roswire/ros2/ros2.py

```diff
@@ -21,25 +19,16 @@ class ROS2:
     app_instance: 'AppInstance'
     nodes: ROS2NodeManager = attr.ib(init=False)
     services: ROS2ServiceManager = attr.ib(init=False)
-    _state_probe: ROS2StateProbe = attr.ib(init=False)
```

Comment:
Why remove the state probe?

Reply:
`StateProbe.for_app_instance()` is not implemented in this branch, so it was giving me an error when I was trying to run test cases for `launch` and `locate`.

Reply:
It still looks like state probe has been removed.